rustls-bench/src/main.rs

```diff
@@ -413,41 +419,32 @@ fn multithreaded(
     })
 }
 
-fn report_handshake_result(
-    variant: &str,
-    params: &BenchmarkParam,
-    clientauth: ClientAuth,
-    resume: ResumptionParam,
-    rounds: u64,
-    timings: Vec<Timings>,
-) {
+fn report_handshake_result(variant: &str, params: &Parameters, rounds: u64, timings: Vec<Timings>) {
     print!(
         "{}\t{:?}\t{:?}\t{:?}\tclient\t{}\t{}\t",
         variant,
-        params.version,
-        params.key_type,
-        params.ciphersuite,
-        if clientauth == ClientAuth::Yes {
-            "mutual"
-        } else {
-            "server-auth"
+        params.proto.version,
+        params.proto.key_type,
+        params.proto.ciphersuite,
+        match params.client_auth {
```

Comment:
Nit: further steps, consider extracting this into a method on `Parameters`?

Reply:
hoisted up the `client_auth` match into a fn on `ClientAuth`